modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -73,6 +73,57 @@ spec:
       rootCA: ""
 ----
 
+* Optional: Customize the route name of the Keycloak instance by adding a host name attribute in the Argo CD CR. Use this feature to support advanced routing use cases such as balancing incoming traffic load among multiple link:https://docs.openshift.com/container-platform/latest/networking/ingress-sharding.html#nw-ingress-sharding_ingress-sharding[Ingress Controller shards].
++
+** To add a `host name` attribute in the Argo CD CR, add this YAML snippet in your namespace.
++
+[source,yaml]
+----
+apiVersion: argoproj.io/v1alpha1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+  labels:
+    example: route
+spec:
+  sso:
+    provider: keycloak
+    keycloak:
+     host: sso.test.example.com #<1>
+  server:
+    ingress:
+      enabled: true
+    insecure: true
+----
+<1> Specifies the name of the host key
++
+** To apply the YAML, run the following command in the {gitops-shortname} CLI:
++
+[source,terminal]
+----
+kubectl apply -f argocd.yaml -n <any-namespace>
```

Comment:
I also noticed that this document uses oc commands instead of kubectl: https://docs.openshift.com/gitops/1.13/accesscontrol_usermanagement/configuring-sso-for-argo-cd-using-keycloak.html. If needed, you can update the current PR to use oc commands as well. For example, you can replace kubectl apply -f argocd.yaml -n <any-namespace> with oc apply -f argocd.yaml -n <any-namespace>. Both commands work the same.

Reply:
Fixed.